lang/fr-fr.js

```diff
@@ -68,7 +68,7 @@ export default {
 	"components.input-time-range.startTime": "Heure de début",
 	"components.interactive.instructions": "Appuyer sur entrée pour interagir, sur Echap pour quitter",
 	"components.link.open-in-new-window": "S’ouvre dans une nouvelle fenêtre.",
-	"components.list.keyboard": "Use <b>arrow keys</b> to move focus inside this list, or <b>page up/down</b> to move up or down by 5",
+	"components.list.keyboard": "Utilisez les <b>touches fléchées</b> pour vous concentrer sur cette liste, ou <b>faites défiler la page vers le haut/bas</b> pour la faire défiler de 5 cm",
```

Comment:
Is this `5 cm` thing weird?.... @dlockhart, ever seen anything like this?

Reply:
```suggestion
	"components.list.keyboard": "Utilisez les <b>touches fléchées</b> pour vous concentrer sur cette liste, ou <b>faites défiler la page vers le haut/bas</b> pour la faire défiler de 5",
```

Reply:
Submitted this as an error: [TR-113](https://desire2learn.atlassian.net/servicedesk/customer/portal/38/TR-113)